src/pages/proposals/index.tsx

```diff
@@ -96,6 +109,14 @@ const Proposals: FunctionComponent = () => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	if (!featureFlags) {
```

Comment:
You can do `!featureFlagStore.error && (featureFlagStore.isFetching || !featureFlagStore.response)`
That would be more accurate. It would show the loader page load and if there is an error, it won't display the loader